docs/js/guides/bas-external-system.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Connecting to external system from the Business Application Studio (BAS) with the SAP Cloud SDK for JavaScript / TypeScript
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Connecting to External system form BAS
+description: This article describes how the SDK helps to connect to external systems from the Business Application Studio.
+keywords:
+- sap
+- cloud
+- sdk
+- proxy
+- connectivity
+- Business Application Studio
+- cloud-foundry
+- JavaScript
+- TypeScript
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+## What is the Business Application Studio (BAS)? ##
+
+The Business Application Studio (BAS) is a development environment offered as a service on Cloud Foundry (CF).
+You can simply subscribe to the BAS and from there quickly start developing without installing node, git, Visual Studio Code or other tools.
+From the look and feel it is very similar to Visual Studio Code, which is no surprise since it is based on [Eclipse Theia](https://theia-ide.org/), the open source version of Visual Studio Code.
+
+However, SAP added a few useful features to the BAS. 
+You can connect the BAS to your CF account.
+This allows you to reach behind a Cloud Connector from your locally running BAS application without deploying to CF.
```

Comment:
```suggestion
You can connect the BAS to your CF account, which allows you to reach behind a Cloud Connector from your locally running BAS application without deploying to CF.
```
I as a dummy reader don't 100% how the Cloud Connector comes into play here. I guess I would have to have a Cloud Connector configured (and could normally reach the systems behind the CC from CF), so maybe this would be a better suggestion:
```suggestion
You can connect the BAS to your CF account, which allows you to reach all systems that you have configured on CF from your local machine. This includes systems that are connected through the Cloud Connector and are normally not reachable through the internet.
```

Reply:
like the second version.